pkg/core/object/metadata.go

```diff
@@ -197,3 +200,641 @@ func VerifyHeaderForMetadata(hdr object.Object) error {
 	}
 	return nil
 }
+
+var (
+	// ErrUnreachableQuery is returned when no object ever matches a particular
+	// search query.
+	ErrUnreachableQuery = errors.New("unreachable query")
+	// ErrWrongValOIDDelim is returned when data base stores unexpected
+	// attribute's value delimeter.
+	ErrWrongValOIDDelim = errors.New("wrong value-OID delimiter")
+	// ErrInvalidCursor is returned when [SearchCursor] has invalid seek
+	// values.
+	ErrInvalidCursor = errors.New("invalid cursor")
+
+	errInvalidPrimaryFilter  = errors.New("invalid primary filter")
+	errWrongPrimaryAttribute = errors.New("wrong primary attribute")
+	errWrongKeyValDelim      = errors.New("wrong key-value delimiter")
+
+	// AttributeDelimiter is attribute key and value separator used in metadata DB.
+	AttributeDelimiter = []byte{0x00}
+	metaOIDPrefix      = []byte{metaPrefixID}
+
+	maxUint256 = new(big.Int).SetBytes([]byte{255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+		255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255})
+	maxUint256Neg = new(big.Int).Neg(maxUint256)
```

Comment:
maxNeg sounds a lot like `min` to me.

Reply:
https://github.com/nspcc-dev/neofs-node/blob/092ff01ee240397eda24d76105bb9c89760742f6/pkg/local_object_storage/metabase/metadata.go#L44